release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2819,3 +2819,23 @@ link:https://access.redhat.com/solutions/6954628[{product-title} 4.10.11 contain
 ==== Updating
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-10-12"]
+=== RHBA-2022:1601 - {product-title} 4.10.12 bug fix and security update
+
+Issued: 2022-05-02
+
+{product-title} release 4.10.12, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1601[RHBA-2022:1601] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1600[RHSA-2022:1600] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6955942[{product-title} 4.10.12 container image list]
+
+[id="ocp-4-10-12-bug-fixes"]
+==== Bug fixes
+* Previously, the Infrastructure Operator could not provision X11- and X12-based systems due to validation errors created by the bare metal controller (BMC) when special characters such as question marks or equal signs were used in the `filename` parameters of URLs. With this update, the `filename` parameter is removed from the URL if the virtual media image is backed by a local file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011626[*BZ#2011626*])
```

Comment:
You might want to break the first sentence into two. Possibly,
```suggestion
* Previously, the Infrastructure Operator could not provision X11- and X12-based systems. This was due to validation errors created by the bare metal controller (BMC) when special characters such as question marks or equal signs were used in the `filename` parameters of URLs. With this update, the `filename` parameter is removed from the URL if the virtual media image is backed by a local file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011626[*BZ#2011626*])
```
But I am not sure I like that suggestion. You could probably come up with something better. :)

Reply:
I'm okay with it. I have incorporated your suggestion.